Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -138,8 +138,42 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
         /// <returns>The project relative path.</returns>
         /// <remarks>This doesn't produce paths that contain step out '..' relative paths.</remarks>
         public static string GetAssetDatabasePath(string absolutePath)
```

Comment:
Please add tests for this function.

I think that dealing with Path.GetFullPath is going to be tricky (aside from creating temp files in tests) but the other stuff would be important to have test coverage.

The string manipulation below is "simple" but easy to regress, and I can't imagine how scary it would be to make a change in the future and hope that we don't break one of the three clauses.

Reply:
Added